extensions/amp-a4a/rtc-documentation.md

```diff
@@ -196,6 +196,143 @@ In both cases, the requirements for an errorReportingUrl are the same:
 
 The error ping will be sent by creating an image pixel in the document. See `sendErrorMessage` in [real-time-config-manager.js](https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/0.1/real-time-config-manager.js) for implementation details.
 
+### AMP Consent Integration
+
+By default, if the consent state of the page is not `SUFFICIENT` or `UNKNOWN_NOT_REQUIRED`, then no RTC callouts will be sent from the page. However, you may optionally modify this setting, to whitelist specific RTC callouts that should be sent regardless of the consent state. You can modify this across all RTC requests for a given ad slot or on a per-RTC-callout basis. You also may either whitelist for all consent states, or only specific consent states, using the RTC Config attribute `sendRegardlessOfConsentState`. 
+
+The value of `sendRegardlessOfConsentState` should either be `true` or an array of consent policy states as defined in src/consent-state.js. If set to true, then all consent states are treated as valid for sending RTC. If set to an array, then only the values in that array (in addition to the defualt values of `SUFFICIENT` and `UNKNOWN_NOT_REQUIRED`) are treated as valid.
```

Comment:
Is it the string true or the boolean true?  Does it matter?

Can you change "If set to true, then all consent states are treated as valid for sending RTC." to "In the RTC request would normally be surpressed, it will be sent if sendRegardlessOfConsentState is set to true or the consent state is within the array of states given"?

Reply:
done